compiler/tests/success/slh/x86-64/annot.jazz

```diff
@@ -0,0 +1,14 @@
+fn f(#transient reg u64 x) -> #secret reg u64 {
```

Comment:
Do I understand correctly that these tests are for the checker and not for the lowering of SSLH? (We never reach that pass because there are not export functions.)

Reply:
Fixed.